### PR TITLE
Update to JupyterLite 0.4.0

### PR DESCRIPTION
**Pull request by jtpio**
Follow-up to https://github.com/jupyterlite/jupyterlite/releases/tag/v0.4.0a0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@
 
 | status | `jupyterlite-pyodide-kernel` | `jupyterlite-core` |  `jupyterlab`  |   `notebook`   |  `retrolab`  |
 | :----: | :--------------------------: | :----------------: | :------------: | :------------: | :----------: |
+|  pre   |           `0.4.*`            |    `>=0.4,<0.5`    | `>=4.2.0,<4.3` | `>=7.2.0,<7.3` |      -       |
```
Comment (bollwyvl) [inline README.md:26]: maybe take this opportunity to add the pyodide/python versions to the matrix?

Comment (jtpio) [reply README.md:26]: Done in https://github.com/jupyterlite/pyodide-kernel/pull/106/commits/cf7d410031adf862099b4d44b94f95e694f49339

**Review by bollwyvl — APPROVED** [review]
Runs fine on RTD :tada:

**Comment by jtpio** [comment]
Thanks for the help!

**Pull request merged**